ang/civicase/case/services/cases-utils.service.js

```diff
@@ -17,11 +17,29 @@
       ContactsCache.add(contacts);
     };
 
+    /**
+     * Process all case clients contacts ids.
+     *
+     * @param {Array} contacts of a case.
+     * @returns {Array} of all client ids.
+     */
+    this.getAllCaseClientContactIdsFromAllContacts = function (contacts) {
```

Comment:
`getAllCaseClientContactIdsFromAllContacts` can we try to simplify this name. Maybe just `getAllCaseClientContactIds`? 
(its a minor comment, change only if you agree)